Support for the `--output` flag (#2490)
kyma-project/cli: docs/extensibility/actions.md

| Name | Type | Description |
| --- | --- | --- |
| **dryRun** | bool | Simulates resource deletion if set to `true` |

> | **output** | enum | Changes the output format if not empty. One of `yaml` or `json` |

Reviewer: ```suggestion
| **output** | enum | Changes the output format if not empty. It can be `.yaml` or `.json`. |
```

| Name | Type | Description |
| --- | --- | --- |

> | **output** | enum | Changes the output format if not empty. One of `yaml` or `json` |

Reviewer: ```suggestion
| **output** | enum | Changes the output format if not empty. It can be `.yaml` or `.json`. |
```
